server/chat/background/prediscovery_task.py

```
@@ -238,6 +240,9 @@ def _should_run_for_user(user_id: str) -> bool:
     from utils.db.connection_pool import db_pool
     from datetime import datetime, timedelta
 
+    if not get_user_preference(user_id, "prediscovery_enabled", True):
```

Ruff (0.15.11)

[warning] Boolean positional value in function call

(FBT003)